apps/auth/models.py

```diff
@@ -31,3 +32,16 @@ def __init__(self, username: str, email: str, password=None):
 
     def set_password(self, password):
         self.password = generate_password_hash(password)
+
+    def get_last_10_quiz_attempts(self):
```

Comment:
I would change name of this method to `def get_latest_quiz_attempts(self, number_of_quizes: int=10)`
And modify function behavior too use passed param and return expected number of quizzes, this will make this method more reusable